src/bin/substudy.rs

```diff
@@ -21,7 +21,7 @@ Subtitle processing tools for students of foreign languages
 
 Usage: substudy clean <subs>
        substudy combine <foreign-subs> <native-subs>
-       substudy export csv <video> <foreign-subs> [<native-subs>]
+       substudy export csv [--limit=<secs>] <video> <foreign-subs> [<native-subs>]
        substudy export review <video> <foreign-subs> [<native-subs>]
        substudy export tracks <video> <foreign-subs>
```

Comment:
I didn't particularly like the name "limit" either, but I couldn't think of a better one at the time. "End-at" is much better, thanks! I've changed it to "end-at" throughout.